### PR TITLE
Update eslint packages

### PR DESCRIPTION
**Pull request by koddsson**
1. Upgraded the eslint packages
2. Ran eslint with `--fix`
3. Wrote a little script to walk the files where there are violations still after running `--fix` and adds comments to disable the violating rules. This way the new rules can be brought in slowly without the need for me to hastily refactoring right now.

Here's the script I wrote:

```js
/* eslint-disable prefer-const */
const fs = require('fs')
const getStdin = require('get-stdin')

getStdin().then((input) => {
  const lines = input.split('\n').filter(line => line !== '')
  let filename = null
  let rulesApplied = []
  lines.forEach((line) => {
    if (line.startsWith('/')) {
      filename = line
      rulesApplied = []
    } else {
      const data = line.split(' ').filter(word => word !== '')
      const rule = data[data.length - 1]
      if (!rulesApplied.includes(rule)) {
        const fileContents = fs.readFileSync(filename)
        const newFile = `/* eslint-disable ${rule} */\n${fileContents.toString()}`
        fs.writeFileSync(filename, newFile)
        rulesApplied.push(rule)
      }
    }
  })
})
```

### CODE REVIEW

File: endpoints/horses/index.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-continue */
+/* eslint-disable no-mixed-operators */
+/* eslint-disable no-plusplus */
```
Comment (koddsson) [inline endpoints/horses/index.js]: `eslint` fails locally for me on this rule in this file but I have no idea why 🤔

**Comment by koddsson** [comment]
So it seems that [codeship isn't running on forked repos any more](https://documentation.codeship.com/general/projects/getting-started/#testing-prs-from-forked-repositories) and that's why I'm not getting the build status here? That seems like a weird regression.

**Comment by koddsson** [comment]
Now CircleCI is running all the tests even tho it's set to run only the lint job? 🤔

**Comment by koddsson** [comment]
Nope. Seems to be still failing, not sure why. Digging into the CircleCI documentation now.

**Comment by koddsson** [comment]
This seems to have been a issue with the force pushing. I noticed it in the CircleCI UI when checking the configuration that it didn't detect the configuration. Pushing a "empty" commit fixed it.

**Comment by MiniGod** [comment]
By the way, Codeship never had a feature to build forked repos.

**Comment by koddsson** [comment]
>By the way, Codeship never had a feature to build forked repos.

Oh dang, how do you know that PRs from contributors are valid then? 🙆‍♂️

**Pull request merged**